CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v1.5.2
+
+* Enhancements
+  * Added `NERVES_DATA_DIR` environment variable for specifying the location
+    for global artifacts and downloads.
+  * Configure `NERVES_DATA_DIR` to `XDG_DATA_HOME` if defined.
+  * Add `nerves` config for `source_date_epoc` to enable reproducible builds.
```

Comment:
* Nerves tooling now supports setting the `SOURCE_DATE_EPOCH` environment variable for reproducible builds during compilation via `:source_date_epoch` in your application config. This removes timestamp differences between builds. See [reproducible-builds.org](https://reproducible-builds.org/) for more information.